### ISSUE TITLE
Support partials with parameters

### ISSUE THREAD

**Issue by idmontie** [XaminProject/handlebars.php#128]
HandlerbarsJS has supported passing parameters to partials (see [this test](https://github.com/wycats/handlebars.js/blob/e290ec24f131f89ddf2c6aeb707a4884d41c3c6d/spec/partials.js#L26-L32) and [this one](https://github.com/wycats/handlebars.js/blob/ce74c36118ffed1779889d97e6a2a1028ae61510/spec/qunit_spec.js#L456-L462) from the HandlebarsJS repo ).

All of the following should be supported to match HandlebarsJS 2+:

```handlebars
{{> partial this}}
{{> partial variable}}
{{> partial newVariable=variable}}
```

**Comment by JustBlackBird** [comment]
This is already fixed in #127

**Comment by idmontie** [comment]
Great, feel free to close!
